projects/libreoffice/fonts.conf

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<fontconfig>
+  <dir>/out/instdir/share/fonts/truetype</dir>
```

Comment:
generating a conf on the fly is trivially done, is the output hierarchy flattened in the true deployment vs the docker demo case ?